<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="WL01" failures="0" tests="4" name="com.avengers.bus.services.test.EmailServiceImplTest" time="0.327" errors="1" timestamp="2023-07-02T23:01:05 GST" skipped="3">
  <testcase classname="com.avengers.bus.services.test.EmailServiceImplTest" name="setUp" time="0.327">
    <error message="
Cannot mock/spy class javax.mail.Session
Mockito cannot mock/spy because :
 - final class" type="org.mockito.exceptions.base.MockitoException">
      <![CDATA[org.mockito.exceptions.base.MockitoException: 
Cannot mock/spy class javax.mail.Session
Mockito cannot mock/spy because :
 - final class
at com.avengers.bus.services.test.EmailServiceImplTest.setUp(EmailServiceImplTest.java:33)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:65)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:381)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:319)
at org.testng.internal.invokers.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:178)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:122)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:808)
at org.testng.TestRunner.run(TestRunner.java:603)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:429)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:423)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:383)
at org.testng.SuiteRunner.run(SuiteRunner.java:326)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1249)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
at org.testng.TestNG.runSuites(TestNG.java:1092)
at org.testng.TestNG.run(TestNG.java:1060)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- setUp -->
  <system-out/>
  <testcase classname="com.avengers.bus.services.test.EmailServiceImplTest" name="testGenerateOTP" time="0.000">
    <skipped/>
  </testcase> <!-- testGenerateOTP -->
  <system-out/>
  <testcase classname="com.avengers.bus.services.test.EmailServiceImplTest" name="testSendEmail" time="0.000">
    <skipped/>
  </testcase> <!-- testSendEmail -->
  <system-out/>
  <testcase classname="com.avengers.bus.services.test.EmailServiceImplTest" name="testSendmail" time="0.000">
    <skipped/>
  </testcase> <!-- testSendmail -->
  <system-out/>
</testsuite> <!-- com.avengers.bus.services.test.EmailServiceImplTest -->
